src/pages/clouddocs/annotator_smoking.md

```diff
@@ -0,0 +1,89 @@
+---
+title: "Smoking (Deprecated)"
+excerpt: "."
+categories: Annotators
+slug: annotator_smoking
+toc: true
+---
+<!-- ---
+
+copyright:
+  years: 2011, 2021
+lastupdated: "2019-09-21"
+
+keywords: annotator clinical data, clinical data, annotation
+
+subcollection: wh-acd
+
+---
+
+# Smoking (Deprecated) -->
+
+The smoking annotator identifies whether the patient is a current or former smoker. Furthermore, it identifies the amount and substance(s) being smoked as well as the rate and duration of the smoking habit.
+
+## Configurations
+
+| Configuration | Values | Description |
+|:--------------|--------|-------------|
+| Library | <ul><li>umls.latest</li><li>umls.2021AA</li><li>umls.2020AA</li><li>umls.2019AA <i>(deprecated - will be removed in 2022)</i></li></ul> | Defines the version of the UMLS library that is used when analyzing unstructured data. |
+
+The value `umls.latest` will reference the latest available version of UMLS within the service. As newer versions of UMLS are made available in the service, `umls.latest` library configurations will automatically leverage the latest available version of UMLS in the service once available. Declaration of a specific version of UMLS is recommended to avoid undesirable changes in output as newer versions of UMLS are made available within the service. Through declaration of a specific version of UMLS, newer versions of UMLS may be evaluated prior to use in production.
+
+### Annotation Types
+
+* aci.SmokingInd
+
+##$ aci.SmokingInd
```

Comment:
Presentation/formatting issue with this one?

Reply:
resolved